### PR TITLE
dts: stm32: clean device tree for STM32MP13

### PR DESCRIPTION
**Pull request by GseoC**
Unused peripherals shouldn't be supported at board level for STM32MP13.

### CODE REVIEW

File: core/arch/arm/dts/stm32mp13-pinctrl.dtsi [hunk 1]
```diff
@@ -44,16 +44,4 @@
 		};
 	};
```
Comment (etienne-lms) [inline core/arch/arm/dts/stm32mp13-pinctrl.dtsi:46]: remove also the above empty line

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`

File: core/arch/arm/dts/stm32mp13-pinctrl.dtsi (hunk 1 above)
Comment (GseoC) [reply core/arch/arm/dts/stm32mp13-pinctrl.dtsi:46]: Yep, missed that

**Comment by GseoC** [comment]
Updated and tag applied

**Review by etienne-lms — APPROVED** [review]

**Pull request merged**